opensearchpy/exceptions.py

```diff
@@ -163,6 +169,22 @@ class AuthorizationException(TransportError):
     """Exception representing a 403 status code."""
 
 
+class OpenSearchDslException(Exception):
+    """OpenSearchDsl Exception"""
+
+
+class UnknownDslObject(OpenSearchDslException):
+    """OpenSearchDsl Exception"""
```

Comment:
These exception messages need to be updated.

Reply:
Made the change suggested.